text/0000-new-maintainer-group-process.md

```diff
@@ -0,0 +1,22 @@
+# Codify the Creation of New Maintainer Groups
+
+## Summary
+
+This RFC is intended to define processes for the creation of new maintainer groups.
+
+## Motivation
+
+Currently the creation of new maintainer groups is an ad hoc process that has to be hashed out every time a new group is requested. This means that intervention from the Steering Committee is often needed in situations where they should not need to weigh in. It also means that there is no standard record for the introduction of new groups meaning that the reasoning behind the formation of these new maintainer groups is lost. Codifying this process should make it easier to identify when new maintainer groups should be formed and make the process of doing so much easier. It should also give approval power to the stakeholders that will be directly affected by the introduction of new maintainer groups, namely the existing maintainer groups as opposed to the Steering Committee.
+
+## Implementation
+
+The following are the 3 scenarios in which a new maintainer group would be formed and the procedures for forming each group type.
+
+### New Maintainer Group for New Repositories
+When proposing a new project that the proposer feels does not fall under the domain of an existing maintainer group they can suggest creating a new maintainer group. As part of the proposal, members of the initial maintainer group should be suggested or a request for volunteers should be open by the proposer of the new project. This group will then be approved by the Steering Committee to be the initial set of maintainers.
```

Comment:
```suggestion
When proposing a new project that the proposer feels does not fall under the domain of an existing maintainer group they can suggest creating a new maintainer group. As part of the proposal, members of the initial maintainer group should be suggested or a request for volunteers should be open by the proposer of the new project. This group and the proposed initial set of members will then be reviewed and either approved or rejected by the Steering Committee.
```

Reply:
I think that makes sense to me. In my head that was implied but I think it is fair to lay that out.